deployables/app/app/routes/tokens/balances/balances.tsx

```diff
@@ -65,19 +64,23 @@ const Balances = () => {
               usdValue,
               amount,
               symbol,
+              balanceDiffUsd,
+              balanceDiff,
             }) => (
               <TableRow key={contractId} className="group">
                 <TableCell>
                   <Token logo={logoUrl}>{name}</Token>
                 </TableCell>
                 <TableCell align="right">
-                  <TokenValue symbol={symbol}>{amount}</TokenValue>
+                  <TokenValue symbol={symbol} balanceDiff={balanceDiff}>
+                    {amount}
+                  </TokenValue>
```

Comment:
I explain this a little later. But I think that instead of extending the `TokenValue` and `NumberValue` components I'd rather keep them focussed as they are and compose them with another component like `Diff` that could render it's children based on the diff value and set the text color. The children would then inherit that color (and we'd see the diff immediately instead of only on hover).

```suggestion
                  <TokenValue symbol={symbol}>
                    {amount}
                  </TokenValue>
                  
                  {balanceDiff && <Diff value={balanceDiff}><TokenValue>{balanceDiff}</TokenValue></Diff> }
```